scripts/run-gtest.ps1

```diff
@@ -264,7 +264,7 @@ function Start-TestExecutable([String]$Arguments, [String]$OutputDir) {
         } elseif ($CodeCoverage) {
             $CoverageOutput = Join-Path $OutputDir $CoverageName
             $pinfo.FileName = "C:\Program Files\OpenCppCoverage\OpenCppCoverage.exe"
-            $pinfo.Arguments = "--modules=$(Split-Path $Path -Parent) --cover_children --sources src\core --sources src\inc --sources src\platform --excluded_sources unittest --working_dir $($OutputDir) --export_type binary:$($CoverageOutput) -- $($Path) $($Arguments)"
```

Comment:
Why shouldn't we keep coverage for platform?  We should make sure that changes in platform are still covered by the unittests.

Reply:
I want to start with a smaller scope here. Scoping to just the core also provides a simpler view of code coverage by default. With platform included, we're getting duplication of these files for ever test application, which bloats the coverage gaps. I don't want to enable platform until we can eliminate that bloat.